lib/ex_unit/lib/ex_unit/cli_formatter.ex

```diff
@@ -42,7 +44,8 @@ defmodule ExUnit.CLIFormatter do
     else
       IO.write success(".", config)
     end
-    {:noreply, %{config | test_counter: update_test_counter(config.test_counter, test)}}
+    {:noreply, %{config | test_counter: update_test_counter(config.test_counter, test),
+                          test_timings: update_test_timings(config.test_timings, test)}}
```

Comment:
I considered that, but thought it would muddle the output. There aren't any colors applied to the slow test reports, so you could have passing and failing tests interspersed. If that's an acceptable trade off we can definitely consider failing tests.